<launch>
	<arg name="agent_id" default="1" />

	<node pkg="sunray_swarm" name="rmtt_circle_$(arg agent_id)" type="rmtt_circle" output="screen">
		<param name="agent_id" value="$(arg agent_id)" />
		<param name="desired_yaw" value="0.0" />
		<param name="circle_radius" value="1.0" />
		<param name="linear_vel" value="0.2" />
	</node>
</launch>